pennylane/vqe/vqe.py

```diff
@@ -104,6 +104,15 @@ def terms(self):
         """
         return self.coeffs, self.ops
 
+    @property
+    def wires(self):
+        r"""The sorted union of wires from all operators.
+
+        Returns:
+            (Wires): Combined wires present in all terms, sorted.
+        """
+        return qml.wires.Wires.all_wires([op.wires for op in self.ops], sort=True)
+
```

Comment:
Just to understand correctly, you want it sorted to be unique for VQE problems where the list of observables are permuted, right?

Reply:
Exactly. And I also worry about (though not sure if it can happen) wires in each Tensors where things commute.